sk0-front/src/app/auth/store/auth.effects.ts

```diff
@@ -69,6 +69,11 @@ export class AuthEffects {
       mergeMap((action) =>
         this.authService.register(action).pipe(
           map((response) => registerSuccess(response)),
+          tap(async () => {
```

Comment:
We should handle it in `handleRegisterSuccessEffects$` we already have it in dev branch. You should also refactor the test and the e2e for it.

Reply:
You're right, I thought I already did that. Should be good now.